doc/development/adding_operators.rst

```diff
@@ -70,9 +70,73 @@ The basic components of operators are the following:
      >>> op.terms()
      ((1.0, 2.0), [PauliX(wires=[0]), PauliZ(wires=[0])])
 
-   * Representation via the **eigenvalue decomposition** specified by eigenvalues (for the diagonal matrix, :meth:`.Operator.get_eigvals`)
+   * Representation via the **eigenvalue decomposition** specified by eigenvalues (for the diagonal matrix, :meth:`.Operator.eigvals`)
      and diagonalizing gates (for the unitaries :meth:`.Operator.diagonalizing_gates`):
 
+               >>> op = qml.PauliX(0)
+               >>> op.diagonalizing_gates()
+               [Hadamard(wires=[0])]
+               >>> op.eigvals()
+               [ 1 -1]
+
+
+
+               >>> op = qml.PauliX(0)
+               >>> op.diagonalizing_gates()
+               [Hadamard(wires=[0])]
+               >>> op.eigvals()
+               [ 1 -1]
+
+
+
+          >>> op = qml.PauliX(0)
+          >>> op.diagonalizing_gates()
+          [Hadamard(wires=[0])]
+          >>> op.eigvals()
+          [ 1 -1]
+
+
+
+               >>> op = qml.PauliX(0)
+               >>> op.diagonalizing_gates()
+               [Hadamard(wires=[0])]
+               >>> op.eigvals()
+               [ 1 -1]
+
+
+
+               >>> op = qml.PauliX(0)
+               >>> op.diagonalizing_gates()
+               [Hadamard(wires=[0])]
+               >>> op.eigvals()
+               [ 1 -1]
+
+
+
+          >>> op = qml.PauliX(0)
+          >>> op.diagonalizing_gates()
+          [Hadamard(wires=[0])]
+          >>> op.eigvals()
+          [ 1 -1]
+
+
+
+          >>> op = qml.PauliX(0)
+          >>> op.diagonalizing_gates()
+          [Hadamard(wires=[0])]
+          >>> op.eigvals()
+          [ 1 -1]
+
+
+
+          >>> op = qml.PauliX(0)
+          >>> op.diagonalizing_gates()
+          [Hadamard(wires=[0])]
+          >>> op.eigvals()
+          [ 1 -1]
+
+
+
```

Comment:
I am not sure why these sections show as "added", when only the ``>>> op.eigvals()`` line was changed...